skyplane/api/impl/tracker.py

```diff
@@ -203,3 +212,17 @@ def query_bytes_remaining(self):
             )
         logger.fs.debug(f"[TransferProgressTracker] Bytes remaining per job: {bytes_remaining_per_job}")
         return sum(bytes_remaining_per_job.values())
+
+    def calculate_size(self):
+        if len(self.job_chunk_requests) == 0:
+            return 0
+        bytes_total_per_job = {}
+        for job_uuid in self.job_complete_chunk_ids.keys():
+            bytes_total_per_job[job_uuid] = sum(
+                [
+                    cr.chunk.chunk_length_bytes
+                    for cr in self.job_chunk_requests[job_uuid]
+                    if cr.chunk.chunk_id in self.job_complete_chunk_ids[job_uuid]
+                ]
+            )
+        return sum(bytes_total_per_job.values()) / (2**30)
```

Comment:
Use the GB constant: https://github.com/skyplane-project/skyplane/blob/main/skyplane/utils/definitions.py#L7